### PR TITLE
Added "buildrequires" tasks (bsc#1020033)

### PR DESCRIPTION
**Pull request by lslezak**
To easily install the package build dependencies required for development. 
Related to https://bugzilla.suse.com/show_bug.cgi?id=1020033

- 0.2.18

## The Use Case

```sh
git clone git@github.com:yast/yast-x11.git
cd yast-x11
make -f Makefile.cvs
# so far everything OK
make
...
testX.c:44:10: fatal error: X11/Xlib.h: No such file or directory
# WTF?! What I need to install?!
```

```sh
rake buildrequires:install
sudo zypper install autoconf automake gcc-c\+\+ libtool xorg-x11-libX11-devel xorg-x11-libXmu-devel yast2-devtools\ \>\=\ 3.1.10
....
make
# builds OK
```

### CODE REVIEW

File: lib/tasks/buildrequires.rake [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+#
+#   You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#++
+
+require "shellwords"
+require "open3"
+
+namespace :buildrequires do
```
Comment (jreidinger) [inline lib/tasks/buildrequires.rake]: I just prefer to call it `:build_dependencies` buildrequires is rpm specific. Otherwise it looks good.

Comment (lslezak) [reply lib/tasks/buildrequires.rake]: OK, makes sense.

**Comment by jreidinger** [comment]
When I think about it, two ideas:

1. add it to packaging_rake_tasks as it is not useful only for yast
2. replace hardcoded "package" with https://github.com/openSUSE/packaging_rake_tasks/blob/master/lib/packaging/configuration.rb#L39

**Comment by lslezak** [comment]
Moved to https://github.com/openSUSE/packaging_rake_tasks/pull/41

**Pull request closed without merging**